cli/src/pcluster/aws/ec2.py

```diff
@@ -381,18 +398,33 @@ def get_ebs_snapshot_info(self, ebs_snapshot_id):
     @AWSExceptionHandler.handle_client_exception
     def describe_security_group(self, security_group_id):
         """Describe a single security group."""
-        return self.describe_security_groups([security_group_id]).get("SecurityGroups")[0]
+        return self.describe_security_groups([security_group_id])[0]
 
     @AWSExceptionHandler.handle_client_exception
     def describe_security_groups(self, security_group_ids):
         """Describe a single security group."""
-        return self._client.describe_security_groups(GroupIds=security_group_ids)
+        result = []
```

Comment:
the comment of the function is wrong (it's not your fault).

Reply:
Fixed